src/smartthings/smartapps/smartthings/amazon-echo-home-skill.src/amazon-echo-home-skill.groovy

```diff
@@ -124,7 +131,8 @@ preferences(oauthPage: "oauthPage") {
 			paragraph title: "Or choose individual devices below", ""
 			input "switches", "capability.switch", title: "My Switches", multiple: true, required: false
 			input "thermostats", "capability.thermostat", title: "My Thermostats", multiple: true, required: false
-			input "routinesEnabled", type: "enum", title: "Enable Routines?", options: [[(true): "Yes"]], defaultValue: false, multiple: false, required: false
+			paragraph title: "Note, executing routines via Alexa will not affect SmartHomeMonitor, locks or garagedoors)", "ddsfdsfdsfdsfdsfds"
```

Comment:
trailing ')' after garage doors - not sure if entire string should be in parenthesis, or if this instance should not have parenthesis.